doc/changelog/07-commands-and-options/12358-redirect_printing_params.rst

```diff
@@ -0,0 +1,5 @@
+- **Changed:**
+  Redirect now obeys `Set Printing Width` and associated
```

Comment:
I think this will result in slightly nicer formatting
```suggestion
  :cmdv:`Redirect` now obeys the :flag:`Printing Width` flag and associated
```

Reply:
@JasonGross I've slightly amended your suggestion because the way to refer to commands is through `:cmd:`, not `:cmdv:` (the latter does not exist).

Reply:
Done thanks.